providers/configure_server.rb

```diff
@@ -187,6 +187,19 @@
     admin_port admin_port
     secure false
   end
+
+  if not node['install']['tmp_directory'].eql?("")
+    glassfish_jvm_options "JvmOptions java.io.tmpdir" do
+      domain_name domain_name
+      admin_port admin_port
+      username username
+      password_file password_file
+      target target_config
+      secure false
+      options ["-Djava.io.tmpdir=#{node['install']['tmp_directory']}/glassfish"]
```

Comment:
Do we need to create the dir?

Reply:
The `tmp_directory` is provided and Payara will create the `glassfish` sub-directory